src/library/Uno.Themes.WinUI.Markup/Theme/PipsPager.cs

```diff
@@ -0,0 +1,247 @@
+							public static ResourceValue<Brush> Default => new("PipsPagerNavigationButtonForeground", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "PipsPagerNavigationButtonForegroundPointerOver")]
+							public static ResourceValue<Brush> PointerOver => new("PipsPagerNavigationButtonForegroundPointerOver", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "PipsPagerNavigationButtonForegroundPressed")]
+							public static ResourceValue<Brush> Pressed => new("PipsPagerNavigationButtonForegroundPressed", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "PipsPagerNavigationButtonForegroundDisabled")]
+							public static ResourceValue<Brush> Disabled => new("PipsPagerNavigationButtonForegroundDisabled", true);
+						}
+
+						public static class VisualStateEllipse
+						{
+							public static class Height
+							{
+								[ResourceKeyDefinition(typeof(double), "MaterialPipsPagerNavigationButtonVisualStateEllipseHeight")]
+								public static ResourceValue<double> Default => new("MaterialPipsPagerNavigationButtonVisualStateEllipseHeight", true);
+							}
+
+							public static class Width
+							{
+								[ResourceKeyDefinition(typeof(double), "MaterialPipsPagerNavigationButtonVisualStateEllipseWidth")]
+								public static ResourceValue<double> Default => new("MaterialPipsPagerNavigationButtonVisualStateEllipseWidth", true);
+							}
+
+							public static class Fill
+							{
+								[ResourceKeyDefinition(typeof(Brush), "MaterialPipsPagerNavigationButtonVisualStateEllipseFill")]
+								public static ResourceValue<Brush> Default => new("MaterialPipsPagerNavigationButtonVisualStateEllipseFill", true);
+
+								[ResourceKeyDefinition(typeof(Brush), "MaterialPipsPagerNavigationButtonVisualStateEllipseFillPointerOver")]
+								public static ResourceValue<Brush> PointerOver => new("MaterialPipsPagerNavigationButtonVisualStateEllipseFillPointerOver", true);
+
+								[ResourceKeyDefinition(typeof(Brush), "MaterialPipsPagerNavigationButtonVisualStateEllipseFillPressed")]
+								public static ResourceValue<Brush> Pressed => new("MaterialPipsPagerNavigationButtonVisualStateEllipseFillPressed", true);
+
+								[ResourceKeyDefinition(typeof(Brush), "MaterialPipsPagerNavigationButtonVisualStateEllipseFillFocused")]
+								public static ResourceValue<Brush> Focused => new("MaterialPipsPagerNavigationButtonVisualStateEllipseFillFocused", true);
+
+								[ResourceKeyDefinition(typeof(Brush), "MaterialPipsPagerNavigationButtonVisualStateEllipseFillUnfocused")]
+								public static ResourceValue<Brush> Unfocused => new("MaterialPipsPagerNavigationButtonVisualStateEllipseFillUnfocused", true);
+							}
+						}
+
+						public static class PreviousPageData
+						{
+							[ResourceKeyDefinition(typeof(Geometry), "PipsPagerPreviousPageButtonData")]
+							public static ResourceValue<Geometry> Default => new("PipsPagerPreviousPageButtonData", true);
+						}
```

Comment:
```suggestion
							[ResourceKeyDefinition(typeof(Geometry), "PipsPagerPreviousPageButtonData")]
							public static ResourceValue<Geometry> PreviousPageData => new("PipsPagerPreviousPageButtonData", true);
```

Also the same for `NextPageData`, etc.